### ISSUE TITLE
jdk10 installed in /tmp

### ISSUE THREAD

**Issue by amrsoll** [tatsujb/installFAFscript#20]
```bash
        # Download & install java 10 open jdk
        echo "Java 10 installation procedure..."
        to_log "T1 Java 10 installing..."
        wget https://download.java.net/java/GA/jdk10/10.0.2/19aef61b38124481863b1413dce1855f/13/openjdk-10.0.2_linux-x64_bin.tar.gz
        pv $work_dir/faf/openjdk-10.0.2_linux-x64_bin.tar.gz | tar xzp -C $work_dir/faf
        rm openjdk-10.0.2_linux-x64_bin.tar.gz
        echo "" >> $HOME/.bashrc
        echo "" >> $HOME/.bashrc
        ! grep -q 'INSTALL4J_JAVA_HOME' $HOME/.bashrc > /dev/null && echo "export INSTALL4J_JAVA_HOME=$HOME/faf/jdk-10.0.2" >> $HOME/.bashrc
```
The wrong place for the export is set + you cannot simply install the jdk in `/tmp/faf`. Suggesting `~/.local/lib/` or `~/.local/share/`.

**Comment by amrsoll** [comment]
Ok, my bad, the whole tmp/faf dir is moved to $HOME, but it would be best not to have any ~/faf directory at all. I would still go for installing the jdk directly in one of the `.local` folders.

**Comment by tatsujb** [comment]
yeah I'm fine with such a change. I know noone wants that jdk in /usr/local/java/ because Jdk SE 10 is a crapy deprecated version. so it has to be kind of hidden somwhere and my reasoning behind putting it stright inside the faf folder is that no other app will be using it.

at first when I thought that jdk 10 needed to be declared and recognised by the OS, I had a whole bunch of code to determine if you'd already installed, it, what version it was, prompt the user if he wanted it replaced or not, yada yada. now that I know it only needs to have a variable exported in bashrc to work, I cobine that with the idea that this particular version is considered deprecated to determine that it should be as descreet as possible.
